common/src/limits.rs

```diff
@@ -11,22 +11,26 @@ pub struct Limits {
     project_limit: u32,
     /// Whether this user has permission to provision RDS instances.
     rds_access: bool,
+    /// The quantity of RDS instances this user can provision.
+    rds_quota: u32,
 }
 
 impl Default for Limits {
     fn default() -> Self {
         Self {
             project_limit: MAX_PROJECTS_DEFAULT,
             rds_access: false,
+            rds_quota: 0,
```

Comment:
We should be able to remove `rds_access` right?

Reply:
I was worried we would break something if we remove a field, but perhaps that's not the case. :thinking: I'm thinking that the deployers/runtimes will deserialize a claim with a new field just fine, since the new field will be ignored, but if an existing field is removed, they will break.

Reply:
I found that only provisioner uses this field via the `Claim` since provisioner and auth will update at the same time (and we expect little traffic on this route), I went ahead and removed it :sweat_smile:

Reply:
Alright! This is the line I was worried about: https://github.com/shuttle-hq/shuttle/blob/d9ad0179a5d3b3ff0194b81c1558e181addcf592/common/src/claims.rs#L372. The claim layer is used in the runtimes, do you think this will break? It's not entirely clear to me, since it's checking the type id.

Reply:
Ohhhh right :man_facepalming:

Reply:
If it was using serde it would be breaking, but since it's using a type map, I'm not sure if it is breaking. I'm not sure how the TypeId is calculated. :thinking: